### ISSUE TITLE
Could 'write' always use the longest possible prefix?

### ISSUE THREAD

**Issue by sgoetz-brox** [atextor/owl-cli#7]
`test.ttl`:

```
@prefix ex: 	<http://example.com/> .
@prefix ex1: 	<http://example.com/level1/> .

<http://example.com/level1/Hello> a <http://example.com/Test> .
```

`owl write test.ttl` generates:

```
@prefix ex: <http://example.com/> .
@prefix ex1: <http://example.com/level1/> .

ex:level1\/Hello a ex:Test .
```
(changing the order of the prefix declarations in `test.ttl` doesn’t help)

I think this output would be preferable:

```
ex1:Hello a ex:Test .
```

**Comment by atextor** [comment]
Hi @sgoetz-brox,
this is technically not a bug for a change :) but I'd agree, the longest prefix should of course be used.

**Comment by atextor** [comment]
Fixed in [v1.2.3](https://github.com/atextor/owl-cli/releases/tag/v1.2.3).
